pkgs/applications/kde/k3b.nix

```diff
@@ -15,7 +15,7 @@ mkDerivation {
     maintainers = with maintainers; [ sander phreedom ];
     platforms = platforms.linux;
   };
-  nativeBuildInputs = [ extra-cmake-modules kdoctools makeWrapper shared_mime_info ];
```

Comment:
Why is `shared_mime_info` removed from `nativeBuildInputs` but retained in the arguments to the package (above)?

Reply:
it's still present in propagatedBuildInputs